app/models/article.rb

```diff
@@ -1,27 +1,46 @@
 class Article < ApplicationRecord
+
   include PgSearch
   multisearchable :against => [:title, :summary, :content]
+
   extend FriendlyId
   friendly_id :title, use: :slugged
+
+  paginates_per 10
+
   belongs_to :section, optional: true
 
   has_many :authorships
   has_many :contributors,
            through: :authorships,
            dependent: :destroy,
            source: :user
-
   has_many :media, dependent: :destroy
   has_many :comments, dependent: :destroy
   has_many :outquotes, dependent: :destroy
 
   def init
     self.update(is_published: false)
+
+    if self.summary.to_s.empty?
+        preview = self.content.split(' ')[0, 25].join(' ') + '...'
```

Comment:
What's with the to_s?

Reply:
`to_s` is to string. If the summary is nil, it will became an empty string.

Reply:
why not self.summary.nil?

Reply:
Just simpler/easier to do it one line. Why have two cases when you can cover both in one command?

Reply:
No, like why are you converting to int then checking if it's empty when you can just check when it's nil

Reply:
Fine. Your solution is more readable.